src/generic_solver.cpp

```diff
@@ -349,40 +349,7 @@ std::string GenericSolver::to_smtlib_def(Term term) const
 }
 
 Sort GenericSolver::make_sort(const Sort & sort_con, const SortVec & sorts) const {
-  // When constructing a sort, the sort constructor
-  // must have been already processed
-  assert(sort_name_map->find(sort_con) != sort_name_map->end());
-  // The arity of the sort constructor must match
-  // the number of arguments
-  assert(sort_con->get_arity() == sorts.size());
-
-  // All the input sorts must have been processed
-  for (Sort sort : sorts)
-  {
-    assert(sort_name_map->find(sort) != sort_name_map->end());
-  }
-
-  // creating the new sort
-  Sort sort = make_uninterpreted_generic_sort(sort_con, sorts);
-
-  // note that there is no need to communicate anything
-  // to the solver yet. When the sort will be used,
-  // we will print the right name to the solver.
-
-  // assigning a name to the new sort
-  string name = sort->get_uninterpreted_name();
-
-  // store in the map if missing, and return the new sort
-  if (name_sort_map->find(name) != name_sort_map->end())
-  {
-    return (*name_sort_map)[name];
-  }
-  else
-  {
-    (*name_sort_map)[name] = sort;
-    (*sort_name_map)[sort] = name;
-    return sort;
-  }
+  assert(false);
```

Comment:
Is this going to be implemented later or not at all? If it's the latter could you throw an exception instead?